content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'Compares two strings in a case-insensitive manner.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+  - 'path/computer-science'
+---
+
+The **`.equalsIgnoreCase()`** method compares two strings ignoring lower/upper case differences and returns true if the two are the same. Otherwise, it returns false.
+
+## Syntax
+
+```pseudo
+boolean result = stringOne.equalsIgnoreCase(stringTwo);
+```
+
+- `result` is a [boolean](https://www.codecademy.com/resources/docs/general/boolean) type [variable](https://www.codecademy.com/resources/docs/java/variables) that stores the output.
+- `stringOne` and `stringTwo` are the two strings being compared.
+
+## Example
+
+In the example below, there are three different strings named `firstString`, `secondString`, and `thirdString`. To store the result, two boolean type variables are created, which are called `firstResult` and `secondResult`. With the `.equalsIgnoreCase()` method, the values assigned to strings are compared. As a result, there is an output of true, if the strings compared have the same values, and an output of false if the values assigned to those strings are different. Then, as mentioned in the beginning, those values are stored in two different boolean type variables and the result is printed.
+
+```java
```

Comment:
Please add the class and main function:
```
public class CompareStrings {
  public static void main(String[] args) { 
    // Place your code here
  }
}
```
This way it could be compiled as is if a reader wants to follow along with their own compiler. Also, there are a few mistakes with the syntax of your code, I've added corrections below: